docs/source/pages/BW2098POE.rst

```diff
@@ -34,6 +34,7 @@ Key features
 * On-board PCIe ref clk for BW2099 and PCIe/Ethernet bridge
 * User-selectable BW2099 boot configuration switches
 * Design files produced with Altium Designer 20
+* :ref:`PoE injector <Powerig PoE devices>` is required to power the device.
```

Comment:
Small type here on the section name.  Should be `Powering PoE devices`.  (The `n` is missing.)